_posts/2021-08-26-faas-storage-cassandra-astra.md

```diff
@@ -411,6 +430,8 @@ module.exports = async (event, context) => {
 }
 ```
 
+Note that when performing a `find()` operation with the Astra SDK, you will receive a 404 error if the collection is empty or hasn't been created yet. In this instance, the function returns an empty set of results.
```

Comment:
You get a 404 only if the collection does not exist, if it's just empty no errors are thrown. So "is empty" can be removed in this line.

Reply:
Got it, thanks.